src/components/CommentVotingButton.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+
+const CommentVotingButton = ({ comment, setCommentVotes, commentVotes }) => {
+  //   const [thisComment, setThisComment] = useState(comment);
+  const [err, setErr] = useState(null);
+  const handleCommentVoteClick = (event) => {
+    setErr(null);
+    setCommentVotes(comment.votes++);
+    event.currentTarget.disabled = true;
+    setCommentVotes((comment) => {
+      return { ...comment, votes: comment.votes + 1 };
+      // }).catch((err) => {
+      //   setErr("Oops! Something went wrong, please try again later.");
+      //   setThisComment((comment) => {
+      //     return { ...comment, votes: comment.votes - 1 };
+      //   });
```

Comment:
Could you remove your commented out code. Its the worst thing to leave in as no one else will remove it, because they think someone else probably needs it